Add first part of bottom sheet docs (#1203)
telerik/maui-docs: controls/bottomsheet/swipe-gesture.md

---
title: Swipe Gesture
page_title: .NET MAUI BottomSheet Documentation - Swipe Gesture

> description: Learn more about how to disable the swipe gesture the Telerik UI for .NET MAUI BottomSheet control.

Reviewer: ```suggestion
description: Learn how to enable or disable swipe gestures for the Telerik UI for .NET MAUI BottomSheet control to control user interaction and accessibility.
```

position: 3
slug: bottomsheet-swipe-gestrue
---

# .NET MAUI BottomSheet Swiping

> The BottomSheet control for .NET MAUI allows you to use the swipe gesture for opening the sheet view.

Reviewer: ```suggestion
The BottomSheet control supports intuitive swipe gestures that allow users to open, close, and resize the sheet by dragging up or down on the content area. This touch-friendly interaction provides a natural way for users to control the BottomSheet's visibility and state, enhancing the overall user experience on mobile devices.
```

> You can disable the swipe gesture by setting the `IsSwipeEnabled` (`bool`) property. The default value is `true`.

Reviewer: ```suggestion
You can disable the swipe gesture by setting the `IsSwipeEnabled` (`bool`) property. The default value is `true`.

## When to Disable Swipe Gestures

Consider disabling swipe gestures when your BottomSheet contains content that conflicts with the gesture, such as image galleries, horizontal carousels, or interactive maps. For example, if your BottomSheet displays a photo gallery where users swipe left and right to browse images, disabling the vertical swipe gesture prevents accidental dismissal when users are navigating through photos.
```
